pysid/__init__.py

```diff
@@ -15,3 +15,4 @@
 from pysid.identification import *
 from pysid.correlation import *
 from pysid.io import *
+from pysid.interface import *
```

Comment:
@nycolasRamires this module is not present anymore in pysid package, it should be removed.